draft-ietf-quic-manageability.md

```diff
@@ -1083,54 +1083,66 @@ and {{?RFC4443}}.
+packets within a network segment should drop such packets rather than
+fragmenting them. Network operators who plan to implement a more
+selective policy may start by focusing on QUIC.
 
-Networks with configurations that would lead to fragmentation of large packets
-should drop such packets rather than fragmenting them. Network operators who
-plan to implement a more selective policy may start by focusing on QUIC.
 QUIC flows cannot always be easily distinguished from other UDP traffic, but
 we assume at least some portion of QUIC traffic can be identified
-(see {{sec-identifying}}). For QUIC endpoints using DPLPMTUD it is recommended
-for the path to drop a packet larger than the supported size. A QUIC probe
-packet is used to discover the PMTU. If lost, this does not impact the flow of
+(see {{sec-identifying}}). For networks supporting QUIC, it is recommended
+that a path drops any packet larger than the fragmentation size.
+When a QUIC endpoint uses DPLPMTUD, it will use a QUIC probe packet to
+discover the PMTU. If this probe is lost, it will not impact the flow of
 QUIC data.
 
 IPv4 routers generate an ICMP message when a packet is dropped because the
 link MTU was exceeded. {{?RFC8504}} specifies how an IPv6 node generates an
 ICMPv6 Packet Too Big message (PTB) in this case. PMTUD relies upon an
-endpoint receiving such PTB messages {{RFC8201}}, whereas DPLPMTUD does not
+endpoint receiving such PTB messages {{?RFC8201}}, whereas DPLPMTUD does not
 reply upon these messages, but still can optionally use these to improve
 performance {{Section 4.6 of DPLPMTUD}}.
 
-Since a network cannot know in advance which discovery method a QUIC endpoint
-is using, it should always send a PTB message in addition to dropping the
+A network cannot know in advance which discovery method is used by a QUIC 
+endpoint, so it should send a PTB message in addition to dropping an 
 oversized packet. A generated PTB message should be compliant with the
-validation requirements of {{Section 14.2.1 of QUIC-TRANSPORT}}, otherwise it
-will be ignored by DPLPMTUD. This will likely provide the right signal for the
-endpoint to keep the packet size small and thereby avoid network fragmentation
-for that flow entirely.
+validation requirements of {{Section 14.2.1 of QUIC-TRANSPORT}}, otherwise
+it will be ignored for PMTU discovery. This provides a signal to the 
+endpoint to prevent the packet size from growing too large, which can 
+entirely avoid network segment fragmentation for that flow.
+
+Endpoints can cache PMTU information, in the IP-layer cache. This short-term
+consistency between the PMTU for flows can help avoid an endpoint using a
+PMTU that is inefficient. The IP cache can also influence the PMTU value of
+other IP flows that use the same path {{?RFC8201}}{{?DPLPMTUD=RFC8899}},
+including IP packets carrying
+protocols other than QUIC. The representation of an IP path is 
+implementation-specific {{?RFC8201}}.
```

Comment:
```suggestion
validation requirements of {{Section 14.2.1 of QUIC-TRANSPORT}}, otherwise
it will be ignored for PMTU discovery. This provides a signal to the
endpoint to prevent the packet size from growing too large, which can
entirely avoid network segment fragmentation for that flow.

Endpoints can cache PMTU information, in the IP-layer cache. This short-term
consistency between the PMTU for flows can help avoid an endpoint using a
PMTU that is inefficient. The IP cache can also influence the PMTU value of
other IP flows that use the same path {{?RFC8201}}{{?RFC8899}},
including IP packets carrying
protocols other than QUIC. The representation of an IP path is
implementation-specific {{?RFC8201}}.
```